deployment/terraform/create.go

```diff
@@ -877,11 +876,6 @@ func (t *Terraform) setupProxyServer(extAgent *ssh.ExtAgent, instance Instance)
 			cacheObjects = "50m"
 			cacheSize = "16g" // Ideally we'd like half of the total server mem. But the mem consumption rarely exceeds 10G
 			// from my tests. So there's no point stretching it further.
-
-			// MM-58179
-			// We are increasing the receive ring buffer size on the network card. This proved to significantly lower packet loss
-			// (and retransmissions) on particularly bursty connections (e.g. websockets).
-			rxQueueSize = 8192
```

Comment:
My only (minor) concern (which I shared already) is that so far we've tested with a fixed value, which feels a bit more stable and reproducible. We don’t really have evidence that using the maximum is always the best choice, since it's ultimately a tradeoff between latency and throughput. 

That said, I don’t think we plan to vertically scale the proxy nodes any further, especially given that we’re taking the DNS-based load balancing route beyond 100k users. Also, surfacing this through logs is a great improvement which helps mitigate that concern to some extent.